FormMPFormPlaceholder.php

```diff
@@ -144,8 +144,8 @@ private function generateTokens(): array
                 continue;
             }
 
-            $summaryToken[] = sprintf('<div class="label">%s</div>', $v['label']);
-            $summaryToken[] = sprintf('<div class="value">%s</div>', $v['value']);
+            $summaryToken[] = sprintf('<div data-ff-name="%s" class="label">%s</div>', $k, $v['label']);
```

Comment:
That could lead to potentially broken HTML, no? Maybe we should standardize `$k` somehow or at least use `htmlspecialchars()`?

Reply:
I just want to be defensive ;)

Reply:
Alles klar ;-)

Reply:
I don't even know what happens if you enter `super <strange &foobar` as form field name.